tests/test_sanity.py

```diff
@@ -308,31 +298,31 @@ def test_node(self):
 
     ensure_dir('fake')
 
-    with env_modify({'EM_IGNORE_SANITY': '1'}):
-      for version, succeed in [('v0.8.0', False),
-                               ('v4.1.0', False),
-                               ('v4.1.1', True),
-                               ('v4.2.3-pre', True),
-                               ('cheez', False)]:
-        print(version, succeed)
-        f = open(self.in_dir('fake', 'nodejs'), 'w')
-        f.write('#!/bin/sh\n')
-        f.write('''if [ $1 = "--version" ]; then
+    for version, succeed in [('v0.8.0', False),
+                             ('v4.1.0', False),
+                             ('v4.1.1', True),
+                             ('v4.2.3-pre', True),
+                             ('cheez', False)]:
+      print(version, succeed)
+      try_delete(SANITY_FILE)
```

Comment:
was this for your own debugging? if not why is it needed now?

Reply:
You mean the `try_delete(SANITY_FILE)`?  

This is how the test ensures that that sanity test will run each time.   I think its more explicit than the somewhat confusingly named `EM_IGNORE_SANITY`.   The intent of this test is to re-run the sanity checks each time around this loop.

Reply:
As a followup I may propose removing `EM_IGNORE_SANITY` completely, but for now this just removes all the current users.

Reply:
In the context of these tests a better name for `EM_IGNORE_SANITY` would be `EM_FORCE_SANITY`

Reply:
I see, thanks, so `EM_IGNORE_SANITY` was why that line was not needed before... yes, this is better I agree.